tools/build_rules/closure/closure_repositories.bzl

```diff
@@ -79,21 +79,21 @@ def closure_repositories():
   native.new_http_archive(
       name = "closure_compiler",
       build_file_content = CLOSURE_COMPILER_BUILD_FILE,
-      sha256 = "e4e0cb49ad21ec26dd47693bdbd48f67aefe2d94fe8d9239312d2bcc74986538",
-      url = "http://dl.google.com/closure-compiler/compiler-20150729.zip",
+      sha256 = "215ba5df026e5d92bda6634463a9c634d38a1aa4b6dab336da5c52e884cbde95",
+      url = "https://dl.google.com/closure-compiler/compiler-latest.zip",
```

Comment:
Good catch. I actually thought about that but then I saw https://github.com/bazelbuild/bazel/blob/master/tools/build_rules/closure/closure_repositories.bzl#L103 :(

Since Closure Compiler releases come more often, we should not use "latest".